website/docs/docs-section/appendix/savings-comparisons.mdx

```diff
@@ -0,0 +1,186 @@
+---
+id: savings-comparisons
+title: Costs & Savings (Beta)
+sidebar_label: Costs & Savings (Beta)
```

Comment:
nitpick:

```suggestion
title: Token Registry V4 (Beta)
sidebar_label: token_registry V4 (Beta)
```

Reply:
Should the `sidebar_label` be the same as the `title`? 'Cos I think the `sidebar_label` will be displayed on the side, right?

> sidebar_label: The text shown in the document sidebar and in the next/previous button for this document. If this field is not present, the document's sidebar_label will default to its title.

Apparently, if they are the same, we can even omit the `sidebar_label`. Didn't know about this. haha

Reply:
Updated the page title in https://github.com/Open-Attestation/documentation-website/pull/128/commits/2e34d6cf63a4055937cab6e562c8254aea2e23b3